src/astro_image_display_api/image_viewer_logic.py

```diff
@@ -129,15 +129,24 @@ def _default_catalog_style(self) -> dict[str, Any]:
             "size": 5,
         }
 
-    def get_stretch(self, image_label: str | None = None) -> BaseStretch:
+    def get_stretch(
```

Comment:
Do `get` stuff really need `**kwargs`? I thought we want flexibility to set things but get should be straightforward? Am I missing something?

Same comment on all the other `get` methods.

Reply:
I wondered the same thing....definitely `get_catalog_names` and `get_image_labels` will get turned into properties without an arguments once this is merged.